htdocs/tools/textmessage.bml

```diff
@@ -39,7 +39,9 @@ body<=
     unless ($user) {
         $ret .= "<?h1 $ML{'.enter.user.head'} h1?>".
             "<?p " . BML::ml( '.enter.user.text3', { sitenameshort => $LJ::SITENAMESHORT } ) . " p?>".
-            "<div style='margin-left: 40px'><form method='get' action='textmessage'>$ML{'.enter.user.input'} <input type='text' size='15' maxlength='15' name='user' /> <input type='submit' value=\"$ML{'.enter.user.submit'}\" /></form></div>".
+            "<div style='margin-left: 40px'><form method='get' action='textmessage'>$ML{'.enter.user.input'} <input type='text' size='25' maxlength='25' name='user' /> <input type='submit' value=\"$ML{'.enter.user.submit'}\" /></form></div>".
+            #NB the maxlength of the username text box should be based on
+            #LJ::USERNAME_MAXLENGTH, not hardcoded here.
```

Comment:
You can use `$LJ::USERNAME_MAXLENGTH` here. This is in a BML code block, so you can just reference it.